### PR TITLE
Embed API Client: Objective-C

### PR DESCRIPTION
**Pull request by sgammon**
This changeset adds an autogenerated API client in Objective-C for the *Embed API*, which allows Embedded Menus and the metadata that specify them to be fetched from native code in iOS.

Native Swift API client code is coming soon. The client is built against API version `v1`.

### CODE REVIEW

File: iOS/ObjC/Embed/GTLREmbedObjects.h [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// NOTE: This file was generated by the ServiceGenerator.
+
+// ----------------------------------------------------------------------------
+// API:
+//   Embed API (embed/v1)
+// Description:
+//   Provides the ability to embed various Bloombox data in 3rd party apps and
+//   websites.
+
+#import <GoogleApiClientForRest/GTLRObject.h>
```
Comment (sgammon) [inline iOS/ObjC/Embed/GTLREmbedObjects.h:10]: built against the `GoogleApiClientForRest` pod

File: iOS/ObjC/Embed/GTLREmbed.h [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+// ----------------------------------------------------------------------------
+// API:
+//   Embed API (embed/v1)
+// Description:
+//   Provides the ability to embed various Bloombox data in 3rd party apps and
+//   websites.
+
+#import "GTLREmbedObjects.h"
+#import "GTLREmbedQuery.h"
+#import "GTLREmbedService.h"
```
Comment (sgammon) [inline iOS/ObjC/Embed/GTLREmbed.h:12]: header file

File: iOS/Embed/GTLRBBEmbed.h [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+// ----------------------------------------------------------------------------
+// API:
+//   Embed API (embed/v1)
+// Description:
+//   Provides the ability to embed various Bloombox data in 3rd party apps and
+//   websites.
+
+#import "GTLRBBEmbedObjects.h"
+#import "GTLRBBEmbedQuery.h"
+#import "GTLRBBEmbedService.h"
```
Comment (sgammon) [inline iOS/Embed/GTLRBBEmbed.h:12]: top-level header file

File: iOS/Embed/GTLRBBEmbedQuery.h [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+/**
+ *  Retrieve a prepared embedded menu view for a given partner location.
+ *
+ *  Method: embed.view
+ *
+ *  Authorization scope(s):
+ *    @c kGTLRAuthScopeBBEmbedPlusMe
+ *    @c kGTLRAuthScopeBBEmbedUserinfoEmail
+ */
+@interface GTLRBBEmbedQuery_View : GTLRBBEmbedQuery
```
Comment (sgammon) [inline iOS/Embed/GTLRBBEmbedQuery.h:86]: example `@interface` for a request - in this case, `embed.view`, or `GTLRBBEmbedQuery_View`

File: iOS/Embed/GTLRBBEmbedQuery.m [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  GTLRBBEmbedQuery_View *query =
+    [[self alloc] initWithPathURITemplate:pathURITemplate
+                               HTTPMethod:nil
+                       pathParameterNames:pathParams];
+  query.partner = partner;
+  query.location = location;
+  query.style = style;
+  query.expectedObjectClass = [GTLRBBEmbed_EmbeddedViewSpec class];
+  query.loggingName = @"embed.view";
+  return query;
```
Comment (sgammon) [inline iOS/Embed/GTLRBBEmbedQuery.m:74]: actual request method code in Obj-C for `embed.view`, or `GTLRBBEmbedQuery_View`

**Review by tyler-porras — APPROVED** [review]

**Comment by tsamartino** [comment]
I hate how obj-c looks

**Comment by tsamartino** [comment]
(Approve)

**Review by RandyStevens — APPROVED** [review]

**Pull request merged**